logprep/metrics/metrics.py

```diff
@@ -249,6 +244,22 @@ def inner(self, *args, **kwargs):  # nosemgrep
 class CounterMetric(Metric):
     """Wrapper for prometheus Counter metric"""
 
+    def get_collector_type(self) -> type[MetricWrapperBase]:
+        return Counter
```

Comment:
.. like this:

```suggestion
    collector_type = Counter
```

What do you think?

- no method overhead
- declarative